tests/utils/configupdater/postgres/postgres.go

```diff
@@ -87,8 +87,9 @@ func (r *ConfigUpdater) Init(props map[string]string) error {
 	md := pgauth.PostgresAuthMetadata{
 		ConnectionString: props["connectionString"],
 		UseAzureAD:       utils.IsTruthy(props["useAzureAD"]),
+		UseAWSIAM:        utils.IsTruthy(props["useAWSIAM"]),
```

Comment:
Here too we should use `metadata.GetMetadataProperty` (the previous code was incorrect to)